### ISSUE TITLE
Allow memory and CPU from the config and also as part of start flags.

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/crc#139]
We need to allow cpu and memory from the config and also as flag options, also need to put a validation function for same.

**Comment by jmazzitelli** [comment]
I mention this over in https://github.com/code-ready/crc/issues/127#issuecomment-493429805 but since memory and CPU instructions are more relevant here, I'll document here as well for completeness (see that link if you want to know how to increase disk space).

Until this issue is implemented, you can increase memory and CPU for the CRC VM using these instructions (these instructions assume the CRC VM is currently running).

**For MEMORY:**

```
# increase memory to 16GB
virsh -c qemu:///system setmaxmem crc 16000000 --config
virsh -c qemu:///system setmem crc 16000000 --config
crc stop
crc start
```

**For CPUS:**

```
# increase virtual CPUs to 5
virsh -c qemu:///system setvcpus crc 5 --maximum --config
virsh -c qemu:///system setvcpus crc 5 --config
crc stop
crc start
```

**Comment by robin-owen** [comment]
For the memory, can't we use shortened values for the maximum? As an example, `16G` as opposed to `16000000`?

**Comment by jmazzitelli** [comment]
> For the memory, can't we use shortened values for the maximum? As an example, `16G` as opposed to `16000000`

Probably, I didn't try it.

**Comment by robin-owen** [comment]
These commands worked for me:

```
$ sudo virsh setmaxmem crc 16G --config
$ sudo virsh setmem crc 16G --config
$ crc stop && crc start
```

**Comment by anjannath** [comment]
Fixed in #143
